test/expect-tests/dune_console/dune_console_tests.ml

```diff
@@ -95,10 +164,12 @@ let%expect_test "Status line clearing with wrapping." =
   let module Console = New () in
   Console.Backend.set Console.Backend.progress;
   test_status_line_clearing_with_wrapping (module Console);
+  escape [%expect.output];
   [%expect
-    "\n\
-     This status line is a problem because of the fact that it is especially long and \
-     therefore will not be cleared \
-     properly.\r                                                                                                                        \
-     \rHere is another\r               \r "]
+    {| 
+This status line is a problem because of the fact that it is especially long and therefore will not be cleared properly.\r
+                                                                                                                        \r
+Here is another\r
+               \r
```

Comment:
It's interesting that we don't wrap status lines no matter what. The issue is that the console will do it for us in most cases and we won't be able to track the number of clears needed.

Reply:
Actually putting a `Pp.hovbox`, gets it wrapping again and we can see the incorrect clearing sequence.